lib/menu-bar/index.tsx

```diff
@@ -48,6 +49,9 @@ export const MenuBar: FunctionComponent<Props> = ({
     ? 'Notes With Selected Tag'
     : 'All Notes';
 
+  const CmdOrCtrl = isMac ? 'Cmd' : 'Ctrl';
+  const title = showTrash ? '' : `New Note • ${CmdOrCtrl}+Shift+I`;
```

Comment:
considering that this title is specific to the new note button I know I would find it more straightforward if we put the value inside of the JSX itself so I don't have to wander in order to find it.

```ts
title={showTrash ? '' : `New Note…`}
```

otherwise if we keep it here let's try picking a more specific name than `title` since there are multiple "title" values in mind with the menu bar. an explanatory comment would help considerably too since my first impressions were that we were doing strange things in this component vs. in its parent, until I realized this has to be this way.

```ts
// when the trash is open we want to disable creating new notes
// but we want the icon to remain, so hide the title to make it disappear
const newNoteTitle = showTrash ? … : …;
```

that being said, is there a reason we need to hide the button title?
the history button has a disabled state that adds "unavailable" to explain why the button is disabled. is there a reason not to just always show the title with or without an explanation? seems like it could be confusing to someone trying to figure out what that button is and why it's disabled.

Reply:
You're right @dmsnell it makes a lot more sense to keep the title in the jsx. When I added the `CmdOrCtrl` I thought at first it may be cleaner to move the title by it as well. 

>that being said, is there a reason we need to hide the button title?

I should have followed the example of the history button. I've done that now leaving a title but showing unavailable. Doing that and moving it back into the jsx I think makes it more understandable without the need for comments, but I can add that as well if you think it would still help.